lstchain/scripts/benchmarks/charge_benchmark.py

```diff
@@ -0,0 +1,105 @@
+
+
+import argparse
+import matplotlib.pyplot as plt
+from lstchain.reco import r0_to_dl1
+from lstchain.io.config import read_configuration_file, get_standard_config
+from lstchain.paths import r0_to_dl1_filename
+import ctaplot
+from ctaplot.plots.calib import plot_charge_resolution, plot_photoelectron_true_reco, plot_pixels_pe_spectrum
+from pathlib import Path
+import logging
+import sys
+import tables
+from astropy.table import Table
+from matplotlib.backends.backend_pdf import PdfPages
+import os
+
+
+
+log = logging.getLogger(__name__)
+
+parser = argparse.ArgumentParser(description="R0 to DL1")
+
+# Required arguments
+parser.add_argument('--input-file', '-f', type=Path,
+                    dest='input_file',
+                    help='Path to the simtelarray file',
+                    required=True
+                    )
+
+# Optional arguments
+parser.add_argument('--output-dir', '-o', action='store', type=Path,
+                    dest='output_dir',
+                    help='Path where to store the reco dl2 events',
+                    default='./benchs_results/')
+
+parser.add_argument('--config', '-c', action='store', type=Path,
+                    dest='config_file',
+                    help='Path to a configuration file. If none is given, a standard configuration is applied',
+                    default=None
+                    )
+
+
+args = parser.parse_args()
+
+
+def main():
+
+    ctaplot.set_style()
```

Comment:
I hope this does not break anything 😄

Reply:
Matplotlib supports context managers for style changes.